src/routes/safe/components/TransactionsNew/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import { currentPageSelector, newTransactionsCurrentPageSelector } from '../../store/selectors/newTransactions'
+import { useFetchNewTransactions } from '../../container/hooks/useFetchNewTransactions'
+import { ButtonLink, Loader } from '@gnosis.pm/safe-react-components'
+import { setPreviousPage } from '../../store/actions/transactionsNew/setPreviousPage'
+import { setNextPage } from '../../store/actions/transactionsNew/setNextPage'
+
+const Transactions = (): React.ReactElement => {
+  const dispatch = useDispatch()
+  const transactions = useSelector(newTransactionsCurrentPageSelector)
+  const { currentPage, maxPages } = useSelector(currentPageSelector)
+  useFetchNewTransactions()
+
+  const nextPageButtonHandler = () => {
+    dispatch(setNextPage())
+  }
+
+  const previousPageButtonHandler = () => {
+    dispatch(setPreviousPage())
```

Comment:
does it have sense? it will be an infinite scroll Table I think we won't ever make use of previousPage.

Reply:
Yes I know, I just put that to check that the pagination works without implementing now an infinite scroll, I suggest to remove is once we implement it

Reply:
can you add a TODO please?

Reply:
done